crates/turborepo-lib/src/task_graph/visitor/exec.rs

```diff
@@ -365,7 +353,7 @@ impl ExecContext {
             }
         }
 
-        let cmd = self.command()?;
+        let cmd = self.cmd.take().expect("should be present");
```

Comment:
Just curious, why is this an `Option`? Do we have other ways of initializing an `ExecContext` that don't include a command?

Reply:
This is because `exec` takes a reference to `&self` instead of `self` so we can't move `cmd` out of `self`. I'll switch this to a `clone` to make it clear this is always present.